runtime/include/chpl-comm.h

```diff
@@ -82,14 +82,14 @@ extern const int chpl_heterogeneous;
 // before the request completes (after waiting on the returned handle)
 chpl_comm_nb_handle_t chpl_comm_get_nb(void* addr, c_nodeid_t node, void* raddr,
                                        size_t size, int32_t typeIndex,
-                                       int ln, c_string fn);
+                                       int ln, int32_t fn);
```

Comment:
It would be nice to have it all the same (I'd prefer without "const") just because over time this sort of one-way-here, another-way-there thing can make software into an ugly mess even while leaving it functionally correct. But at the same time, I don't think it's important enough to hold this PR for. Just try to keep in mind that it should be cleaned up eventually.